XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -465,6 +465,41 @@ protected virtual void OnFormClosing(PropertiesDialogClosingEventArgs e)
 
         private void verticalTabs_SelectedIndexChanged(object sender, EventArgs e)
         {
+            
+            var snapshotTypePageSpecific = verticalTabs.SelectedItem as NewPolicySnapshotTypePageSpecific<VMSS>;
+            if (snapshotTypePageSpecific != null)
+            {
+              
+                switch (newPolicyVMSSTypePage1.BackupType)

```

Comment:
I suggest to move all this logic into the NewPolicySnapshotTypePageSpecific class to keep the specific implementation outside of the PropertiesDialog. For example:
`snapshotTypePageSpecific.ToggleQuiesceCheckBox(newVMSSVMsPage1.SelectedVMs);`